src/API/Management/Jobs.php

```diff
@@ -59,6 +59,40 @@ public function importUsers($file_path, $connection_id, $params = [])
         return $request->call();
     }
 
+
+    /**
+     * Create an export users job.
+     * Required scope: "read:users"
+     *
+     * @param  array  $params
+     * @return mixed
+     */
```

Comment:
Any reason not to link out to the API docs for this endpoint? Since the method just takes an array of parameters that should align with the documentation for this endpoint, would help developers understand what can be passed in.

Reply:
Good call, I'll add those to the functions